OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -41,7 +41,7 @@ public virtual void Initialize(Menu menu)
         public virtual void Initialize(Menu menu, LayoutGroup layoutGroup)
         {
             Menu = menu;
-            _layoutGroup = layoutGroup;
+            this.LayoutGroup = layoutGroup;
```

Comment:
`this.`? is this JS/TS? :P

Reply:
I should probably rename it because class and field having same names seems to confuse VS

Reply:
Ah yeah, that never goes well :P

Reply:
fixed

Reply:
now named 'Layout'

Reply:
also renamed Console to MyConsole

Reply:
How about OwmlConsole? At least it's somewhat descriptive

Reply:
Makes sense